website/content/en/docs/building-operators/helm/migration.md

```diff
@@ -73,12 +73,50 @@ $ operator-sdk create api \
 **Note** Ensure that you use the same values for the flags to recreate the same Helm Chart and API's. If you have
 more than one chart or API's you can add them via `operator-sdk create api` command. For further information check the [quick-start][quickstart]. 
  
-### Replacing the content
+### Migrating your CR's
 
-- Update the CR manifests in `config/samples` with the values of the CR's in your old project which are in `deploy/crds/`
-- Check if you have customizations options in the `watch.yaml` file of your previous project and then, update the new `watch.yaml` file with the same ones
-- Ensure that all roles configured in the `/deploy/roles.yaml` will be applied in the new project in the file `config/rbac/role.yaml`
-- If you have customizations in your `helm-charts` then, apply them in the new `helm-charts`. Note that this directory was not changed at all.
+Update the CR manifests in `config/samples` with the values of the CR's in your old project which are in `deploy/crds/`.
+
+### Migrating your Watches
+Check if you have customizations options in the `watch.yaml` file of your previous project and then, update the new `watch.yaml` file with the same ones. In our example, it will look like:
+
+```yaml
+# Use the 'create api' subcommand to add watches to this file.
+- group: example.com
+  version: v1alpha1
+  kind: Nginx
+  chart: helm-charts/nginx
+# +kubebuilder:scaffold:watch
+```
+
+**NOTE**: Do not remove the [Maker][marker] `+kubebuilder:scaffold:watch`. It allows the tool to update the watches file when new API's are created. 
+
+### Checking the Permissions (RBAC)
+
+SDK generated the roles for you in the `config/rbac/role.yaml`. However, you might need to  
+update this file with the permissions which are in your previous project in `/deploy/roles.yaml`.
+
+And then, beware that:
+
+- By default, the projects are `cluster-scoped`. Check that your role is of the `ClusterRole` Kind. For further information check the [operator-scope][operator-scope] doc.  
+
+- The `apiGroups` for `monitoring.coreos.com` is no longer required. If you would like to export the Prometheus metrics check the next section `Exporting metrics`.
+
+### Migrating your Helm Charts
+
+If you have customizations in your `helm-charts` then, apply them in the new `helm-charts`. Note that this directory was not changed at all.
```

Comment:
But it will create the api based in the chart. However, imagine the scenario:
- The dev creates the project and the api based on chart
- Then start to work on it and then, did customizations directly in the `helm-charts` dir of the project and no longer in the specific chart. Could not that be common?

Reply:
It would be common, but any of those customizations would be copied over automatically if they use `create api --helm-chart=<existing_project>/helm-charts/<chart>`

Reply:
> The scenario that I am trying to cover here is when: the changes were applied directly in the operator project and not in the chart used to gen the API.

Yes that's the scenario that's covered by copying the helm chart from the existing project.